pom.xml

```diff
@@ -176,6 +176,7 @@
     <maven-stylus-skin.version>1.5</maven-stylus-skin.version>
     <maven-surefire-plugin.version>3.0.0-M1</maven-surefire-plugin.version>
     <maven-checkstyle-plugin.version>3.1.2</maven-checkstyle-plugin.version>
+    <maven-assembly-plugin.version>3.3.0</maven-assembly-plugin.version>
```

Comment:
Let's update apache pom version to 27 (the latest) and use the inherited version (i.e. 3.4.1) of `maven-assembly-plugin`.
```
+++ b/pom.xml
@@ -18,7 +18,7 @@
   <parent>
     <groupId>org.apache</groupId>
     <artifactId>apache</artifactId>
-    <version>23</version>
+    <version>27</version>
     <relativePath /> <!-- resolve from repository -->
   </parent>
```

Reply:
Got.